lib/kontena/templates/haproxy/index.text.erb

```diff
@@ -0,0 +1,39 @@
+global
```

Comment:
All the other "partials" are named based on the HAProxy config "context", shoud this file also follow that convention and be named as `global.text.erb`? `index` is IMO bit confusing name :)